user-guide/Advanced_Functionality/Swarming/Messages/SwitchingObjects.md

```diff
@@ -0,0 +1,25 @@
+---
+uid: SwitchingObjects
+---
+
+# Switching objects
+
+With DataMiner Swarming, you can switch certain [DataMiner objects](xref:BasicConcepts#system-components) from one DataMiner Agent in a cluster to another.
+
+> [!NOTE]
+> In the first version of DataMiner Swarming, scheduled for production in Q4 of 2023, it will only be possible to switch elements from one DMA to another. No other objects are available for Swarming yet.
+
```

Comment:
@RobbeDGH, I have added this note to clarify that as of now only elements can be switched. Is this OK?

Reply:
Is OK.